lib/fs.js

```diff
@@ -2978,6 +3002,8 @@ module.exports = fs = {
   ftruncateSync,
   futimes,
   futimesSync,
+  isFile,
+  isDirectory,
```

Comment:
It seems like these should have "sync" as part of their name.

Reply:
Refactoring

Reply:
Done

Reply:
Also, could you say how I could post the comparison benchmarks?